### PR TITLE
Pppsys 48527 improve iframe handler security

### PR DESCRIPTION
**Pull request by pwachala**
the improved iframe handler and fixed PP logo

### CODE REVIEW

File: templates.js [hunk 1]
```diff
@@ -1,1857 +1,1848 @@
...
+        that will be shown as an iframe and use this handler to catch the messages
+        sent from the iframe on the parent website and turn them into events.
+      `,
+      template: `
+  ${fs.readFileSync(path.join(__dirname, 'build/pushToAnalytics.js'), { encoding: 'utf-8' })}   
+  
+  window.addEventListener('message', function(event){ 
+    if (event.origin !== 'https://'+'{{domain}}') {
+      return;
+    }
```
Comment (rezik007) [inline templates.js:1690]: What is decision behind it?

File: webpack.config.js [hunk 2]
```diff
@@ -46,9 +46,12 @@ module.exports = {
           {
             test: /\.(ts|tsx|jsx|js)$/,
             loader: require.resolve('ts-loader'),
+            options: {
+              transpileOnly: true,
+            }
```
Comment (rezik007) [inline webpack.config.js:51]: I saw my previous comments that suggested to add it but right now I see no reason, please remove.

**Review by rezik007 — APPROVED** [review]

**Comment by rezik007** [comment]
Please close [duplicated PR](https://github.com/PiwikPRO/solutions-templates/pull/55) after merge this one.

File: templates.js (hunk 1 above)
Comment (kbigos) [reply templates.js:1690]: https://piwikpro.atlassian.net/browse/PPPSYS-48527

"Any script in a document in a window can request to let a message get dispatched to a document in another window whose window object it has obtained, by calling .postMessage() on that window object. Consequently, any event listener used to receive messages must first check the identity of the sender of the message, using the origin and possibly source properties. This cannot be overstated: Failure to check the origin and possibly source properties enables cross-site scripting attacks."

File: webpack.config.js (hunk 2 above)
Comment (pwachala) [reply webpack.config.js]: removed

**Pull request merged**